frontend/src/layout/onboarding/FrameworkInstructions/go_snippet.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Snippet from './snippet'
+
```

Comment:
This format of splitting snippets is repeated across all frameworks so we could modify this to be way more reusable. However, I manually separated each one for now incase there's some extra information that's needed depending on the framework